ts/features/payments/checkout/screens/WalletPaymentInputNoticeNumberScreen.tsx

```diff
@@ -100,14 +102,24 @@ const WalletPaymentInputNoticeNumberScreen = () => {
                 )}
                 value={inputState.noticeNumberText}
                 icon="docPaymentCode"
-                onChangeText={value =>
+                onChangeText={value => {
+                  const trimmedValue = value.replace(/\s/g, "");
+                  const limitedValue =
+                    trimmedValue.length > MAX_LENGTH_NOTICE_NUMBER
+                      ? trimmedValue.substring(0, MAX_LENGTH_NOTICE_NUMBER)
+                      : trimmedValue;
```

Comment:
What do you think to extract this logic in an util function with related unit test?

Reply:
Good idea!
Addressed into ca1195a531d118515889595840bfacf6c2727803